### ISSUE TITLE
Kaufhinweis vor den Optionen konzipieren und rechtlich bestätigen lassen.

### ISSUE THREAD

**Issue by SarahSauer** [typearea/projectmanagement#88]
Kommentar von Tina im Trello: Der Hinweis, dass der nächste Klick einen zahlungspflichtigen Kauf auslöst, muss so formuliert und platziert sein, dass er rechtlichen Angriffen stand hält.
Spielt da die Banalitätsgrenze eine Rolle? Weil wir nur 10 ct haben möchten?
Ist es rechtlich überhaupt möglich durch gleichzeitige Auswahl und Kauf den 1-Click-Kauf zu unterbieten?

**Comment by KatharinaMarks** [comment]
Nein, die Banalitätsgrenze spielt keine Rolle. Der Leser muss wissen, dass er sich vertraglich verpflichtet, für den weiteren Konsum unserer Texte Geld zu zahlen -- egal in welcher Höhe. 
In wiefern unterbieten?

**Comment by ghost** [comment]
@KatharinaMarks Lass uns da mal ausführlicher drüber reden/skypen.
